service-worker.js

```diff
@@ -0,0 +1,23 @@
+---
+---
+
+importScripts('/scripts/sw-toolbox.js');
+
+const VERSION = '{{ site.time }}';
+
+toolbox.cache.name = "Babel-Cache-" + VERSION;
+toolbox.cache.maxEntries = 150;
+toolbox.cache.maxAgeSeconds = 604800;
```

Comment:
@addyosmani @boopathi @jeffposnick I limited the cache to 150 entries and one week. Every time the website will be update the whole cache will be invalided.